pkg/controller/workloadspread/workloadspread_controller.go

```diff
@@ -354,6 +358,8 @@ func (r *ReconcileWorkloadSpread) groupPod(ws *appsv1alpha1.WorkloadSpread, pods
 
 		if _, exist := podMap[subsetName]; exist {
 			podMap[subsetName] = append(podMap[subsetName], pods[i])
+		} else {
```

Comment:
since getSuitableSubsetNameForPod will return FakeSubsetName if no subset is found, is this else clause nesssary?

Reply:
This `else` is for the situation where the pods' subset was deleted.